core/arch/arm/plat-stm32mp1/stm32mp1_misc.S

```diff
@@ -0,0 +1,30 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
```

Comment:
Minor: file name could be improved. `reset.S` maybe? In any case `stm32mp1_` is redundant since we're already in `plat-stm32mp1`.

Reply:
ok :)